shared/src/simulation/behaviors/transfer.ts

```diff
@@ -0,0 +1,407 @@
+                                            draftState,
+                                            vehicle1.id
+                                        )
+                                );
+
+                            sentVehicles[vehicleType] = 0;
+
+                            for (
+                                let index = 0;
+                                index <
+                                Math.min(
+                                    loadableVehicles?.length ?? 0,
+                                    vehicleAmount
+                                );
+                                index++
+                            ) {
+                                const activityId = nextUUID(draftState);
+                                addActivity(
+                                    simulatedRegion,
+                                    LoadVehicleActivityState.create(
+                                        activityId,
+                                        loadableVehicles![index]!.id,
+                                        event.transferDestinationType,
+                                        event.transferDestinationId,
+                                        {},
+                                        behaviorState.personnelLoadTime
+                                    )
+                                );
+                                loadableVehicles![index]!.occupation =
+                                    cloneDeepMutable(
+                                        LoadOccupation.create(activityId)
+                                    );
+                                sentVehicles[vehicleType]++;
+                            }
+                        }
+                    );
+
+                    // Send RequestReceivedEvent into own region
+
+                    addActivity(
+                        simulatedRegion,
+                        DelayEventActivityState.create(
+                            nextUUID(draftState),
+                            RequestReceivedEvent.create(
+                                sentVehicles,
+                                event.transferDestinationType,
+                                event.transferDestinationId,
+                                event.key
+                            ),
+                            draftState.currentTime
+                        )
+                    );
+
+                    // Send event to destination if it is a simulated region
+
+                    if (
+                        event.transferDestinationType === 'transferPoint' &&
+                        draftState.simulatedRegions[event.transferDestinationId]
+                    ) {
+                        addActivity(
+                            simulatedRegion,
+                            SendRemoteEventActivityState.create(
+                                nextUUID(draftState),
+                                event.transferDestinationId,
```

Comment:
Wrong id (transfer point instead of simulated region) here, too.

Reply:
done